### ISSUE TITLE
Update build procedure

### ISSUE THREAD

**Issue by gothub** [ropensci/redland-bindings#77]
Ensure that the build procedure works from a 'clean' build, including a regeneration of the `src/redland.c` file that involves running `swig.sh`. The make 'clean' target does not remove this, but maybe it should. Other issues to resolve:

- [roxygen2 6.1.0](https://github.com/klutometis/roxygen/blob/9adcc8ecb874a802e6947bcd0bd65ff771704a43/NEWS.md) has changed the processing of user defined roclets, so the redland roclet may need to be updated. Running the command `roxygenize(roclets=c("collate", "rd", "namespace", "mergeNamespace_roclet"))` produces the error: 

```
Error in roclet_process.roclet_mergeNamespace(X[[i]], ...) : 
  unused arguments (blocks = blocks, env = env)
```

- a new version of swig (3.0.12) produces a somewhat different `src/redland.c` than swig 3.0.6, as shown in [this PR](https://github.com/ropensci/redland-bindings/pull/76/files#diff-516de39fcf4d5d6dee49003a7527a37aL8573)

The build will first be updated on macOS, then the Docker based build/tests will be updated, which is described [here](https://github.com/ropensci/redland-bindings/issues/69)

**Comment by mbjones** [comment]
I'm pretty sure that swig.sh is no longer in use -- rather, the swig bindings are generated by running autoconf and make.  I had to remove the redland.c file (and .o and .so), but then the build system rebuilt them for me.

**Comment by gothub** [comment]
Yep, that is correct, swig is called directly from the generated Makefile. I'm removing swig.sh.
